jmix-core/core/src/main/java/io/jmix/core/entity/annotation/CaseConversion.java

```diff
@@ -1,44 +0,0 @@
-/*
- * Copyright 2019 Haulmont.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package io.jmix.core.entity.annotation;
-
-import java.lang.annotation.Retention;
-import java.lang.annotation.RetentionPolicy;
-import java.lang.annotation.Target;
-
-import static java.lang.annotation.ElementType.ANNOTATION_TYPE;
-import static java.lang.annotation.ElementType.CONSTRUCTOR;
-import static java.lang.annotation.ElementType.FIELD;
-import static java.lang.annotation.ElementType.METHOD;
-import static java.lang.annotation.ElementType.PARAMETER;
-import static java.lang.annotation.ElementType.TYPE_USE;
-
-/**
- * Indicates that auto case conversion should be used for text input fields bound with marked entity field.
- */
-
-@Target({ METHOD, FIELD, ANNOTATION_TYPE, CONSTRUCTOR, PARAMETER, TYPE_USE })
-@Retention(RetentionPolicy.RUNTIME)
-@MetaAnnotation
-public @interface CaseConversion {
```

Comment:
Make it Deprecated in 2.0 with indication that it will be removed

Reply:
Got it

Reply:
Fixed by: c0d6457c30abce28773b8060f8080212dc9d76a3